Sources/Tokenizer.swift

```diff
@@ -17,12 +17,12 @@ extension String {
           components[components.count-1] += word
         } else if specialCharacters.contains(word) {
           components[components.count-1] += word
-        } else if word != "(" && word.hasPrefix("(") {
-          components.append("(")
+        } else if word != "(" && word.hasPrefix("(") || word != ")" && word.hasPrefix(")") {
+          components.append(String(word.first!))
```

Comment:
Please avoid force unwraps, especially in `Sources`.

You could:
- use `word[0]`
- unroll the if test into a separate one
- use `prefix(1)`
- ...